README.md

```diff
@@ -166,6 +166,19 @@ RP2040 chip according to how it is connected up on the Trinkey.
 [Adafruit-Trinkey-QT2040]: https://github.com/rp-rs/rp-hal-boards/tree/main/boards/adafruit-trinkey-qt2040
 [adafruit trinkey qt2040]: https://www.adafruit.com/product/5056
 
+### [blok-rp2040] - Board Support for the [Blok microcontroller]
```

Comment:
The naming convention we have for boards here is `vendor-device`, so this project should be named
`boardsource-block` and placed in the `boards/boardsource-block` directory.
You should also refer to the board as `Blok` instead of `Blok microcontroller`, as this is what the vendor calls it most of the time.

Reply:
I've made changes accordingly to the naming convention.